README.rst

```diff
@@ -18,37 +18,45 @@ We hope to give users & applications empowering developer-friendly privacy enhan
 
 
 
-.. image:: https://img.shields.io/pypi/v/aiootp
-    :target: https://img.shields.io/pypi/v/aiootp
-    :alt: version
+.. image:: https://img.shields.io/pypi/v/aiootp?style=flat-square&logo=pypi
+    :target: https://img.shields.io/pypi/v/aiootp?style=flat-square&logo=pypi
+    :alt: aiootp PyPI Package Version
 
-.. image:: https://img.shields.io/pypi/pyversions/aiootp?color=black
-    :target: https://img.shields.io/pypi/pyversions/aiootp?color=black
-    :alt: python-versions
+.. image:: https://img.shields.io/pypi/wheel/aiootp?style=flat-square&color=red&logo=pypi&logoColor=gold
+    :target: https://img.shields.io/pypi/wheel/aiootp?style=flat-square&color=red&logo=pypi&logoColor=gold
+    :alt: Python Wheel Availability Badge
 
-.. image:: https://img.shields.io/badge/License-AGPL%20v3-red.svg
-    :target: https://img.shields.io/badge/License-AGPL%20v3-red.svg
-    :alt: license
+.. image:: https://img.shields.io/pypi/pyversions/aiootp?style=flat-square&color=gold&logo=python
+    :target: https://img.shields.io/pypi/pyversions/aiootp?style=flat-square&color=gold&logo=python
+    :alt: Python Versions Badge
 
-.. image:: https://img.shields.io/badge/code%20style-black-000000.svg
-    :target: https://img.shields.io/badge/code%20style-black-000000.svg
-    :alt: code-style
+.. image:: https://img.shields.io/badge/coverage-99%25-blue?style=flat-square&logo=codecov&logoColor=maroon
+    :target: https://img.shields.io/badge/coverage-99%25-blue?style=flat-square&logo=codecov&logoColor=maroon
+    :alt: Test Coverage Badge
 
-.. image:: https://img.shields.io/pypi/wheel/aiootp
-    :target: https://img.shields.io/pypi/wheel/aiootp
-    :alt: python-wheel-availability
+.. image:: https://img.shields.io/badge/Linter-Ruff-D7FF64?style=flat-square&logo=ruff
+    :target: https://img.shields.io/badge/Linter-Ruff-D7FF64?style=flat-square&logo=ruff
+    :alt: Ruff Linter Badge
 
-.. image:: https://github.com/rmlibre/aiootp/actions/workflows/linux-python-app.yml/badge.svg
-    :target: https://github.com/rmlibre/aiootp/actions/workflows/linux-python-app.yml/badge.svg
-    :alt: linux-build-status
+.. image:: https://img.shields.io/badge/Formatter-Ruff-D7FF64?style=flat-square&logo=ruff
+   :target: https://img.shields.io/badge/Formatter-Ruff-D7FF64?style=flat-square&logo=ruff
+   :alt: Ruff Formatter Badge
 
-.. image:: https://github.com/rmlibre/aiootp/actions/workflows/windows-python-app.yml/badge.svg
-    :target: https://github.com/rmlibre/aiootp/actions/workflows/windows-python-app.yml/badge.svg
-    :alt: windows-build-status
+.. image:: https://img.shields.io/badge/License-AGPL%20v3-purple?style=flat-square&logo=GNU&logoColor=maroon
+    :target: https://img.shields.io/badge/License-AGPL%20v3-purple?style=flat-square&logo=GNU&logoColor=maroon
+    :alt: Gnu Affero General Public License Badge
```

Comment:
These colors can be made into a nice rainbow. Also, the targets should link to relevant places.

Reply:
They look great! The links are better now.

Resolved by (https://github.com/rmlibre/aiootp/pull/12/commits/17f4330955756838430a6f1bb6145f0f8511c253)